db/migrate/20180819185350_add_fields_to_relief_camp.rb

```diff
@@ -0,0 +1,12 @@
+class AddFieldsToReliefCamp < ActiveRecord::Migration[5.2]
+  def change
+    add_column :relief_camps, :latitude, :float
+    add_column :relief_camps, :longitude, :float
+    add_column :relief_camps, :address, :text
+    add_column :relief_camps, :phone_no, :string
```

Comment:
convert strings to text? use model validations to manage constraints?

Reply:
postgresql driver automatically puts it as varchar without a limit. should be alright.